src/Observable.ts

```diff
@@ -17,7 +17,7 @@ export class Observable<T> extends BaseObservable<T> {
         }
 
         if (values[Symbol.iterator]) {
-            return new Observable<T>(({next, error}) => {
+            return new Observable<T>(({next, error, complete}) => {
```

Comment:
w sumie głupoty pisze - tutaj ewidentnie tworzysz observable z immview, wiec przejdzie.

niemniej byłoby czytelniej trzymać to jednym stylu